src/test/java/teammates/test/cases/datatransfer/FeedbackSessionResultsBundleTest.java

```diff
@@ -0,0 +1,59 @@
+package teammates.test.cases.datatransfer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.DataBundle;
+import teammates.common.datatransfer.FeedbackSessionResultsBundle;
+import teammates.common.datatransfer.attributes.FeedbackQuestionAttributes;
+import teammates.common.datatransfer.attributes.FeedbackResponseAttributes;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.logic.core.FeedbackQuestionsLogic;
+import teammates.logic.core.FeedbackResponsesLogic;
+import teammates.logic.core.FeedbackSessionsLogic;
+import teammates.test.cases.logic.BaseLogicTest;
+
+/**
+ * SUT: {@link teammates.common.datatransfer.FeedbackSessionResultsBundle}.
+ */
+public class FeedbackSessionResultsBundleTest extends BaseLogicTest {
```

Comment:
FeedbackSessionResultsBundleTest should belongs to unit test rather than logic test.

In a word, you shall test whether the `getActualResponses` did the correct filter or not by just using some fake data.

Extend `BaseTestCase` is good enough.

Reply:
Let me tell you what I have done. So I have a json file with many questions and answers. I load it into Db and get responses for a question and then match it with expected responses. Is that correct way?

Reply:
Nope. Take a look at `InstructorFeedbackResultsResponseRowTest.java`, though this one is more about the UI side.

Reply:
I looked at `InstructorFeedbackResultsResponseRowTest.java`. So I have to cut the Db part..right? Take fake responses from json to make a bundle, use `getActualResponses` and lastly compare with expected results.

Reply:
Yes. A unit test should use minimal dependencies.